src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -29,7 +29,51 @@
 /**
  * @ApiResource(
  *     collectionOperations={
- *         "get": {"method": "GET"},
+ *         "get": {
+ *             "method": "GET",
+ *             "requirements": {"id": "%pattern_uuid%"},
+ *             "swagger_context": {
+ *                 "parameters": {
+ *                     {
+ *                         "name": "status",
+ *                         "in": "query",
+ *                         "type": "string",
+ *                         "description": "The status of the Idea resource.",
+ *                         "enum": IdeaStatusEnum::ALL_STATUSES,
```

Comment:
Same here, you can use IdeaStatusEnum::toArray().

Reply:
as above: callbacks are not supported here.